react-accordion-test/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "react-accordion-test",
```

Comment:
What's this file?

Reply:
The next PR removes this folder.

We can either merge in the second PR to this one, and then review. Or we can merge this in and then the second one fixes this.

I'm happy either way, but let's make a move on this.

Reply:
Merge it! Merge it! Merge it! :tada: :partying_face: